automation/WinFormsAutomationApp/AuthenticationHelper.cs

```diff
@@ -294,7 +356,17 @@ private static void UpdateCache(KeyValuePair<TokenCacheKey, AuthenticationResult
             TokenCache.DefaultShared.StoreToCache(updated.Value, updated.Key.Authority, updated.Key.Resource, updated.Key.ClientId, updated.Key.TokenSubjectType, new CallState(new Guid()));
             NotifyAfterAccessCache(updated.Key.Resource, updated.Key.ClientId, updated.Value.Result.UserInfo.UniqueId, updated.Value.Result.UserInfo.DisplayableId);
         }
+
+        private static List<KeyValuePair<TokenCacheKey, AuthenticationResultEx>> QueryCache(string authority,
```

Comment:
If it is KeyValuePair, updating cache becomes easier. no need to cast it back to keyvaluepair